stable/mlrun/values.yaml

```diff
@@ -243,9 +243,25 @@ api:
   sidecars:
     logCollector:
       enabled: false
+      # log collection modes:
+      #   - "sidecar" - use the sidecar container to collect logs
+      #   - "best-effort" - use the sidecar container to collect logs if fails fallback to legacy method
+      #   - "legacy" - use the legacy method to collect logs
+      mode: "sidecar"
+
       name: log-collector
       fullnameOverride:
 
+      # chunk size in bytes
+      bufferSizeBytes: 10485760
+      getLogsBufferPoolSize: 512
+      logCollectionbufferPoolSize: 512
+
+      readLogWaitTime: "3s"
+      stateFileUpdateInterval: "10s"
+      monitoringInterval: "30s"
```

Comment:
I know I put 30s as well but I think we should change the default to something faster, maybe `10s` as well